src/style.css

```
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-weight: 300;
+  font-family: Avenir, "Helvetica", sans-serif;
+}
+
+body {
+  margin: 0;
+  width: 100%;
+}
+
+.form {
+  position: relative;
+}
+
+.form::before {
+  content: "";
+  position: absolute;
+  display: inline-block;
+  top: 50%;
+  transform: translateY(-50%);
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.form-big::before {
+  left: 26px;
+  width: 20px;
+  height: 20px;
+}
+
+.form-small::before {
+  left: 13px;
+  background-size: 11px;
+  width: 11px;
+  height: 11px;
+}
+
+.search-bar {
+  display: block;
+  min-width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: inherit;
+  margin-top: 20px;
+}
+
+.search-bar-small {
+  height: 42px;
+  padding: 10px 10px 10px 34px;
```

and same to other search-bar

```suggestion
  paddingLeft: 33px;
```